### ISSUE TITLE
dbeaver 24.2.5 column expression bug using MariaDB 11.6.2

### ISSUE THREAD

**Issue by McVanS** [dbeaver/dbeaver#36366]
### Description

Creating a column that is generated from other columns generates error:

SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'STORED GENERATED NULL' at line 1
You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'STORED GENERATED NULL' at line 1.

I can create the same column using the same expression using HeidiDB with no issues.

### DBeaver Version

24.2.5

### Operating System

Windows 10

### Database and driver

MariaDB 11.6.2
jdbc.mysql driver

### Steps to reproduce

Using HeidiDB create a column with an expression set to preserve using concat(`field1`,', ',`field2`) or script it:

```
CREATE TABLE `test` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `field1` varchar(48) NOT NULL,
  `field2` varchar(50) NOT NULL,
  `fullname` varchar(100) GENERATED ALWAYS AS (concat(`field1`,', ',`field2`)) STORED,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_uca1400_ai_ci;
```

Using DBeaver brows to the table, select the expression column and for example add a space to the expression and try to save the column.

**Comment by ShadelessFox** [comment]
Hello @McVanS,

Can you please provide the exact query that failed to execute and the steps required to replicate the issue?

**Comment by McVanS** [comment]
There was no query. I simply tried to create a column based on an expression that combines two other columns in the same table into one new column using the DBeaver interface. The create table script should explain what I tried to achive except from within DBeaver. I probably should have mentioned that it is an error raised by DBeaver itself but I assumed that would be assumed. Interestingly enough running the script as stated here from within DBeaver creates the table and the expression based column as expected proving that it probably isn't a driver issue but more likely how the interface constructs the query internally. What is also interesting is updating the expression column after it was created with the scrip within DBeaver also raises the same error.
